http-client-core/src/main/java/io/micronaut/http/client/bind/DefaultHttpClientBinderRegistry.java

```diff
@@ -178,6 +168,11 @@ protected DefaultHttpClientBinderRegistry(ConversionService<?> conversionService
         }
     }
 
+    @Override
+    public Optional<AnnotatedClientRequestBinder<?>> findBinder(Class<?> annotationType) {
```

Comment:
Name this something more descriptive since `findBinder` could refer to argument binders or request binders